data/instruct_advanced_sqlite.csv

```diff
@@ -19,7 +19,7 @@ broker,sqlite,instructions_date_join,"SELECT strftime('%Y-%m', sbCustJoinDate) A
 CR = customer rank by total transaction amount, where the top customer has the highest amount.
 PMCS = per month customer signups. PMAT = per month average transaction amount. Truncate date to month for aggregation.
 TAC = Total Active Customers who joined after a specified date"
-broker,sqlite,instructions_date_join,"SELECT COUNT(DISTINCT sb.sbTxId) AS num_transactions, SUM(sb.sbTxAmount) AS total_transaction_amount FROM sbTransaction AS sb JOIN sbCustomer AS sc ON sb.sbTxCustId = sc.sbCustId WHERE LOWER(sc.sbCustCountry) = 'usa' AND sb.sbTxDateTime >= DATE('now', '-7 days', 'weekday 0', '-6 days') AND sb.sbTxDateTime <= DATE('now', '-7 days', 'weekday 0');",How many transactions were made by customers from the USA last week (exclusive of the current week)? Return the number of transactions and total transaction amount.,"Last week = DATE('now', '-7 days', 'weekday 1', '-7 days') to before DATE('now', '-7 days', 'weekday 1'). Always join transactions with customers before using the transactions table.","To analyze stock performance, join the daily price and ticker tables and calculate price change To get the total transaction amount per customer, join the customer and transaction tables, group by customer, and sum the transaction amounts. Last week = DATE('now', '-7 days', 'weekday 1', '-7 days') to before DATE('now', '-7 days', 'weekday 1'). Always join transactions with customers before using the transactions table. To get the success rate of transactions per customer, join customer and transaction tables, group by customer, and calculate the percentage of successful transactions"
+broker,sqlite,instructions_date_join,"SELECT COUNT(DISTINCT sb.sbTxId) AS num_transactions, SUM(sb.sbTxAmount) AS total_transaction_amount FROM sbTransaction AS sb JOIN sbCustomer AS sc ON sb.sbTxCustId = sc.sbCustId WHERE LOWER(sc.sbCustCountry) = 'usa' AND sb.sbTxDateTime >= DATE('now', '-7 days', 'weekday 1', '-7 days') AND sb.sbTxDateTime <= DATE('now', '-7 days', 'weekday 1');",How many transactions were made by customers from the USA last week (exclusive of the current week)? Return the number of transactions and total transaction amount.,"Last week = DATE('now', '-7 days', 'weekday 1', '-7 days') to before DATE('now', '-7 days', 'weekday 1'). Always join transactions with customers before using the transactions table.","To analyze stock performance, join the daily price and ticker tables and calculate price change To get the total transaction amount per customer, join the customer and transaction tables, group by customer, and sum the transaction amounts. Last week = DATE('now', '-7 days', 'weekday 1', '-7 days') to before DATE('now', '-7 days', 'weekday 1'). Always join transactions with customers before using the transactions table. To get the success rate of transactions per customer, join customer and transaction tables, group by customer, and calculate the percentage of successful transactions"
```

Comment:
Whoops sorry for not matching the query to the instructions even though they are semantically similar. Good catch 🙏